templates/web/pages/bmviii_home_temp.tmpl

```diff
@@ -0,0 +1,265 @@
+            <div class="bmviii-left-accordion-content">
+                <p>
+                    You can have a maximum of 4 people on your team.
+                </p>
+            </div>
+        </div>
+        <div class="column is-half"; style = "padding-top 120px;">
+            <button class="bmviii-right-accordion">What are the requirements to attend?</button>
+            <div class="bmviii-right-accordion-content">
+                <p>
+                    BoilerMake is open to all undergraduate college students who are of age 18 or older.
+                </p>
+            </div>
+            <button class="bmviii-right-accordion">My team was accepted and I wasn't - can I attend?</button>
+            <div class="bmviii-right-accordion-content">
+                <p>
+                    Feel free to email team@boilermake.org for info about our waitlist, and we will do our best to accommodate you as space opens up at our event.
+                </p>
+            </div>
+            <button class="bmviii-right-accordion">Do you allow walk-ins and last minute registration?</button>
+            <div class="bmviii-right-accordion-content">
+                <p>
+                    We accept walk-ins based on availability after hacker check-in.
+                    We will announce more details closer to the event!
+                </p>
+            </div>
+            <button class="bmviii-right-accordion">I missed the deadline to apply - can I attend?</button>
+            <div class="bmviii-right-accordion-content">
+                <p>
+                    We set a deadline to ensure that all applications are given equal consideration to attend BoilerMake.
+                    Feel free to email team@boilermake.org for info about our waitlist, and we will do our best to accommodate you as space opens up at our event.
+                </p>
+            </div>
+            <button class="bmviii-right-accordion">What if I have any other questions?</button>
+            <div class="bmviii-right-accordion-content">
+                <p>
+                    Please email us at <a href='mailto:team@boilermake.org'>team@boilermake.org</a>!
+                </p>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="section has-background-image" id="testimonials" style='background-image: url({{ static_path "bmviii_testimonial_full_background.svg" }})'>
+    <div class="container">
+        <div class="container" style="padding: 0 1.5rem;">
+                <div class="columns is-centered content">
+                    <div class="column is-2 bmviii-section-header" style="margin-left: 48px; margin-right: 12px; margin-top:72px">
+                        <img alt="Testimonial" src={{ static_path "bmviii_testimonial_text.svg" }}>
+                    </div>
+                </div>
+        </div>
+        <div class="carousel-wrapper" style="margin-top: 125px; margin-bottom: 250px">
```

Comment:
Carousel is cool, good work! But it seems like there's some bugs in it:
![Screen Shot 2020-10-05 at 21 20 39](https://user-images.githubusercontent.com/10781907/95148329-a445a700-0748-11eb-9792-94f4d22130b0.png)
The towel background thing repeats but only partially

Also, maybe we can do something about the typography? The alternating red then white make reading the text hard.

Reply:
I need to fix up the background for the carousel. I think design gave the ok to make the text in a white background